src/hooks/useTasks.js

```diff
@@ -84,7 +87,7 @@ function useOpenTasks(tasks, now, courtSettings) {
     }
     return openTasks
     // Since we are using our own generated cache key we don't need to add the convertedTasks to the dependency list.
-  }, [convertedTasksPhasesKey]) // eslint-disable-line react-hooks/exhaustive-deps
+  }, [convertedTasksPhasesKey, voidedDisputes]) // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
`voidedDisputes` isn't tracked by React, so it doesn't need to be in the dependency list.

Reply:
mmhh added with the auto-save, let me check